### PR TITLE
Post Detail Screen

### PR DESCRIPTION
**Pull request by sidcr7-likeminds**
Jira ticket - https://likemindscommunity.atlassian.net/browse/LM-7838

### CODE REVIEW

File: app/src/main/java/com/likeminds/feedsx/post/detail/view/adapter/databinder/ItemPostDetailReplyViewDataBinder.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+            tvCommenterName.text = data.user.name
+            tvCommentContent.text = data.text
+
+            if (data.isLiked) ivLike.setImageResource(R.drawable.ic_like_comment_filled)
+            else ivLike.setImageResource(R.drawable.ic_like_comment_unfilled)
+
+            tvCommentTime.text = TimeUtil.getDaysHoursOrMinutes(data.createdAt)
+
+            if (data.likesCount == 0) likesCount.hide()
```
Comment (ishaan1607) [inline app/src/main/java/com/likeminds/feedsx/post/detail/view/adapter/databinder/ItemPostDetailReplyViewDataBinder.kt:68]: write if-else in proper paranthesis, it improves code readablity

File: app/src/main/java/com/likeminds/feedsx/posttypes/util/PostTypeUtil.kt [hunk 2]
```diff
@@ -300,34 +275,112 @@ object PostTypeUtil {
...
-                seeLessClickableSpan,
+
+        val postTextClickableSpan = object : ClickableSpan() {
+            override fun onClick(p0: View) {
+                adapterListener.postDetail(data)
+            }
+
+            override fun updateDrawState(ds: TextPaint) {
+                ds.isUnderlineText = false
+            }
```
Comment (ishaan1607) [inline app/src/main/java/com/likeminds/feedsx/posttypes/util/PostTypeUtil.kt:293]: what is `p0` and `ds`? name proper argument names

File: app/src/main/java/com/likeminds/feedsx/posttypes/util/PostTypeUtil.kt [hunk 3]
```diff
@@ -300,34 +275,112 @@ object PostTypeUtil {
...
+        }
+
+        tvPostContent.post {
```
Comment (ishaan1607) [inline app/src/main/java/com/likeminds/feedsx/posttypes/util/PostTypeUtil.kt:296]: Add comments, what does this `tvPostContent.post{}`?

File: app/src/main/java/com/likeminds/feedsx/posttypes/view/adapter/PostAdapter.kt [hunk 4]
```diff
@@ -51,9 +51,10 @@ class PostAdapter constructor(
         fun savePost() {}
         fun likePost() {}
         fun sharePost() {}
-        fun comment() {}
+        fun comment(postData: PostViewData)
```
Comment (ishaan1607) [inline app/src/main/java/com/likeminds/feedsx/posttypes/view/adapter/PostAdapter.kt:54]: Why we need complete `postData` in comment?

File: app/src/main/java/com/likeminds/feedsx/utils/SeeMoreUtil.kt [hunk 5]
```diff
@@ -1,17 +1,39 @@
...
      * This function is for getting short post content for the see more feature
      */
-    fun getShortContent(postContent: String?, seeMoreCountLimit: Int): String? {
-        if (postContent == null)
+    fun getShortContent(
+        textContent: String?,
+        textView: TextView,
+        maxLines: Int,
+        seeMoreCountLimit: Int
+    ): String? {
```
Comment (ishaan1607) [inline app/src/main/java/com/likeminds/feedsx/utils/SeeMoreUtil.kt:15]: explain what this function does? add single line comments in thr body

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/likeminds/feedsx/posttypes/view/adapter/PostAdapter.kt (hunk 4 above)
Comment (sidcr7-likeminds) [reply app/src/main/java/com/likeminds/feedsx/posttypes/view/adapter/PostAdapter.kt:54]: It is required to fetch the post on the PostDetailScreen. We can also pass postId only. But for testing purposes I was passing postData (as I wanted to show commentsCount)

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**